assets/style.css

```diff
@@ -57,88 +53,96 @@ header h1 {
   border-radius: 8px;
 }
 
-#pickerDiv { 
-    display: flex;
-    flex-wrap: wrap;
-    flex-direction: row; 
-    align-content: space-around;
-    margin: 0 1.5rem 1.5rem 0;
-    width: 30rem;
-    height: 15rem;
+#pickerDiv {
+  display: flex;
+  flex-wrap: wrap;
+  align-content: space-around;
+  margin: 0 1.5rem 1.5rem 0;
+  max-width: 30rem;
+  height: 10.75rem;
 }
 
 .pickBox {
-    height: 2.5rem;
-    width: 2.5rem;
-    border: 1px solid rgb(215, 215, 215);
-    margin: 0.7rem 0.5rem;
-    border-radius: 4px;
+  height: 2.5rem;
+  width: 2.5rem;
+  border: 1px solid rgb(215, 215, 215);
+  margin: 0.25rem 0.5rem;
+  border-radius: 4px;
 }
 
 .navbar {
-    display: inline-flex;
-    flex: wrap;
-    background-color: var(--pale-indigo);
-    color: black;
-    font-size: 1.2rem;
-    font-weight: 500;
-    margin-right: 0.75rem;
-    max-height: 5%;
+  /* display: inline-flex;
+  flex: wrap; */
+  background: linear-gradient(#8a8a8a, #cccccc);
+  /* background-color: var(--aluminum); */
+  color: black;
+  font-size: 1.2rem;
+  font-weight: 500;
+  max-height: 5%;
 }
 
-#navbarBasicExample {
-  background-color: var(--pale-indigo);
+.navbar-item {
+  display: wrap;
+  flex-wrap: wrap;
 }
 
 .navbar-link {
 
-  background-color: var(--pale-indigo);
-  margin-right: 0.75rem;
+  /* background-color: var(--aluminum); */
+  background: linear-gradient(#8a8a8a, #cccccc);
+  /* margin-right: 0.75rem; */
 }
- 
-.navbar-burgar, .navbar-brand, .navbar-dropdown, .navbar-item {
 
-      background-color: var(--pale-indigo);
-    }
-  
-    .navbar-dropdown a {
-      background-color: rgb(255, 255, 255);
-      font-weight: 500;
-    }
+#navbarBasicExample,
```

Comment:
dont override bulma styles

Reply:
You need to include HTML crew in this conversation then.